### ISSUE TITLE
Website Redesign

### ISSUE THREAD

**Issue by matronator** [PrismJS/website#27]
**Motivation**

The website in its current form is kinda ugly (especially the header is a mess that's difficult to navigate visually) and looks a bit outdated.

**Description**

Redesign the website to a cleaner, nicer and more modern style.

**Alternatives**

Not look at the website, ehh?

**Comment by LeaVerou** [comment]
Yup, this is in the cards, we're just focusing on getting v2 out the door asap!

**Comment by rishabhchoudhary1401** [comment]
Hi! I'd like to work on this issue if it's still available.

My initial plan is to:

1. Improve horizontal spacing on mobile for better readability (contents are pushed towards left, need some space there).
2. Move the left navigation sidebar to a more accessible position on smaller screens.
3. Continue with responsive layout and interaction improvements as part of the broader website redesign.

If this scope sounds good, I'd be happy to work on it.

**Comment by LeaVerou** [comment]
Thanks for the willingness to help! We do have a plan for the website already though.

**Comment by rishabhchoudhary1401** [comment]
> Thanks for the willingness to help! We do have a plan for the website already though.

Thanks for replying, anyways if you want me on the term, I'm happy to help.
